agent/ansible/roles/pbench-repo-install/defaults/main.yml

```diff
@@ -1,4 +1,6 @@
 ---
+pbench_repo_url_prefix: https://copr-be.cloud.fedoraproject.org/results/ndokos
```

Comment:
Perhaps make the user in this URL a variable defaulting to `ndokos` so that it can be conveniently changed?

Reply:
I debated this with myself (I'm not sure who won though...) I'd rather treat this as an atomic piece of information: it can be overridden as a whole in the inventory file, but  the default value relieves most people from  having to define it at all. So the onus is put on people who use their own repo (COPR or otherwise) to define it approprirately. WDYT?

Reply:
Perhaps consider:

```
fedoraproject_username: ndokos
pbench_repo_url_prefix: https://copr-be.cloud.fedoraproject.org/results/{{ fedoraproject_username }}
```

That way they can override the `pbench_repo_url_prefix`, or if they are using Fedora Project themselves, they can just change the user name used.

Reply:
Sounds good - I'll push a change.